src/main/java/ch/fhnw/p2p/controller/ProjectController.java

```diff
@@ -107,11 +112,20 @@
 		if (result.hasErrors()) {
 			return new ResponseEntity<HttpStatus>(HttpStatus.PRECONDITION_FAILED);
 		}
+		if (project.getMembers().size() > 0) {
+			User qmUser = userRepo.findByEmail(project.getMembers().iterator().next().getStudent().getEmail()).get();
```

Comment:
Hier müsste theoretisch noch abgefragt werden, ob der User nicht bereits in einem Projekt ist. Also

`userRepo.findByEmailAndStatus(...., User.Status.FREE)`